src/main/cpp/subsystems/Shooter.cpp

```diff
@@ -23,7 +27,7 @@ double Shooter::MotorTopRPM()
   constexpr double TPR {2048};
```

Comment:
Constants are start with `k`. What is TPR? Let's limit abbreviations. Please write out what LTR is.